cmd/snap/cmd_snap_op.go

```diff
@@ -211,11 +211,16 @@ func (mxd mixinDescs) also(m map[string]string) mixinDescs {
 }
 
 var channelDescs = mixinDescs{
-	"channel":   i18n.G("Use this channel instead of stable"),
-	"beta":      i18n.G("Install from the beta channel"),
-	"edge":      i18n.G("Install from the edge channel"),
+	// TRANSLATORS: this should start with an upper case character (if they exist)
+	"channel": i18n.G("Use this channel instead of stable"),
+	// TRANSLATORS: this should start with an upper case character (if they exist)
+	"beta": i18n.G("Install from the beta channel"),
+	// TRANSLATORS: this should start with an upper case character (if they exist)
+	"edge": i18n.G("Install from the edge channel"),
+	// TRANSLATORS: this should start with an upper case character (if they exist)
 	"candidate": i18n.G("Install from the candidate channel"),
-	"stable":    i18n.G("Install from the stable channel"),
+	// TRANSLATORS: this should start with an upper case character (if they exist)
+	"stable": i18n.G("Install from the stable channel"),
```

Comment:
These don't need the `(if they exist)` text for translators. If the channel doesn't contain anything the text just isn't shown (shouldn't be), so the translation will never need to indicate a dual nature.

Reply:
... if *upper case characters* exist.

Reply:
aha. I misunderstood the meaning. In that case then I'm fine with this.

Reply:
This curious why for most it says "this should not start with a lower-case char" but for this one (and some more) it is different?